apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -560,6 +560,7 @@ export async function regenerateQuestion(
   );
 
   const jobData = await serverJob.execute(async (job) => {
+    job.data['questionQid'] = questionQid;
```

Comment:
this is what's being checked for regen, which means that threads couldn't load if it's not there.